operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -1,59 +1,4 @@
 {
-  "__inputs": [
```

Comment:
To be honest, I'm not sure why this was deleted also

Reply:
I'm going to undo this manually, see if the panels render in my dev_setup deployment

Reply:
My first attempt to add this back in failed.

Revisting ....